tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1161,50 +1161,46 @@ describe("Service Form Modal", function() {
         });
 
         it('Should add new set of form fields when "Add Local Volume" link clicked', function() {
+          cy.get("@tabView").contains("Add Local Volume").click();
+
           cy
-            .get("@tabView")
-            .contains(".form-group", "Volume Type")
-            .within(function() {
-              cy
-                .get("select")
-                .should("have.attr", "name", "localVolumes.0.type");
-            });
+            .get('.form-group-heading-content[title="Volume Type"]')
+            .should("have.length", 2);
         });
 
         it('Should add new set of form fields when "Persistent Volume" is selected as volume type', function() {
+          cy.get("@tabView").contains(".form-group", "Volume Type");
+          cy.get("@tabView").find(".button.dropdown-toggle").click();
           cy
-            .get("@tabView")
-            .find('select[name="localVolumes.0.type"]')
-            .select("PERSISTENT");
-
-          // Size input
-          cy
-            .get("@tabView")
-            .find('.form-control[name="localVolumes.0.size"]')
-            .should("exist");
-
-          // Container Path input
+            .contains(".dropdown-select-item-title", "Local Persistent Volume")
+            .click();
+          cy.get('.form-control[name="localVolumes.0.size"]').should("exist");
           cy
-            .get("@tabView")
-            .find('.form-control[name="localVolumes.0.containerPath"]')
+            .get('.form-control[name="localVolumes.0.containerPath"]')
             .should("exist");
         });
 
-        it('Should remove "Volume" form fields when remove button clicked', function() {
-          cy
-            .get("@tabView")
-            .find('select[name="localVolumes.0.type"]')
-            .parents(".panel")
-            .within(function() {
+        it.skip(
```

Comment:
I have added this because the `x` button has disappeared and after 1h or so trying to fix it but since it's already in master I have postponed for maybe another PR.

Reply:
did you try `force`? the button is there, so it should work 🤔 
IMHO having a forced click (with comment) is less worse then having skipped tests…

Reply:
Yes @juliangieseke I thought about adding the force but even though that fixes the tests that won't fix the UI and the integration tests are for the UI.. for that reason I decided to leave a skip instead force otherwise we won't ever really fix.

I'm indifferent adding the `force` though. Just let me know if that is a blocker for you and I can add.

Reply:
hm, no idea… I'd go with `force`, but if you think `skip` is better, thats okay for me. maybe someone else has an opinion on this, it it gets a second approval I'll unblock it :)

Reply:
@juliangieseke yeah I only chose `skip` over `force` because we have been using `force` across the tests so a common pattern. The `skip` is to raise an alert the way alerted you in this PR 😉 

@Fabs this is the case I added the skipped.

Reply:
Since we fix the X button on #2607, and there are already 2 approvals 😄, I will get help with getting green tests and merge today. `ServiceFormModal-cy.js` passes there (running locally), so I believe you will be good to go after rebasing :-).

Reply:
#2607 Merged.